pkg/handler/repositories.go

```diff
@@ -59,12 +36,47 @@ func listRepositories(c echo.Context) error {
 	db.DB.Limit(page.Limit).Offset(page.Offset).Find(&repoConfigs)
 
 	repos := convertToItems(repoConfigs)
-	return c.JSON(200, collectionResponse(&RepositoryCollectionResponse{Data: repos}, c, total))
+	return c.JSON(200, collectionResponse(&api.RepositoryCollectionResponse{Data: repos}, c, total))
+}
+
+// CreateRepository godoc
+// @Summary      Create Repository
+// @ID           createRepository
+// @Description  create a repository
+// @Tags         repositories
+// @Accept       json
+// @Produce      json
```

Comment:
I guess you have to specify @Param  here for each parameter that can be passed?  like this:  https://github.com/swaggo/swag/blob/master/example/celler/controller/examples.go#L126

i don't see an easy way to just reuse the struct, but maybe i'm missing it?